charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,1489 @@ spec:
+                            enabled:
+                              type: boolean
+                            items:
+                              type: array
+                              items:
+                                type: object
+                                required:
+                                - name
+                                - spec
+                                properties:
+                                  name:
+                                    type: string
+                                  spec:
+                                    type: object
+                                    required:
+                                    - prometheusMetricsImporter
+                                    properties:
+                                      prometheusMetricsImporter:
+                                        type: object
+                                        required:
+                                        - query
+                                        properties:
+                                          query:
+                                            type: string
+                        preKube_1_14:
+                          type: object
+                          required:
+                          - items
+                          properties:
+                            items:
+                              type: array
+                              items:
+                                type: object
+                                required:
+                                - name
+                                - spec
+                                properties:
+                                  name:
+                                    type: string
+                                  spec:
+                                    type: object
+                                    required:
+                                    - prometheusMetricsImporter
+                                    properties:
+                                      prometheusMetricsImporter:
+                                        type: object
+                                        required:
+                                        - query
+                                        properties:
+                                          query:
+                                            type: string
+
+            reporting-operator:
+              type: object
+              required:
+              - spec
+              properties:
+                spec:
+                  type: object
+                  properties:
+                    affinity:
+                      type: object
+                    annotations:
+                      type: object
+                    apiService:
+                      type: object
+                      properties:
+                        annotations:
+                          type: object
+                          properties:
```

Comment:
Annotations is just arbitrary key=value annotation pairs, so no explicit properties should be set